node/adapter/datastore.py

```diff
@@ -174,7 +176,8 @@ def from_json(cls, ep_id, json_str):
         json_dict = json.loads(json_str)
         ep = cls(ep_id=ep_id,
                  state=json_dict["state"],
-                 mac=json_dict["mac"])
+                 mac=json_dict["mac"],
+                 if_name=json_dict["container:if_name"])
```

Comment:
Have you thought about the upgrade path here? What happens someone with existing endpoints takes this code. Will it "just work" or will it break. If it breaks we should decide whether to fix it or just release note it.